float32 tau_s
float32 q
float32 tau_df
float32 tau_pf
float32 q_df
float32 q_pf
float32 qp_df
float32 qp_pf
float32 q_bias
float32 dq

